app/assets/javascripts/answers.js

```diff
@@ -20,12 +20,23 @@ $(document).ready(function(e){
   $("#answers").hide();
   $("#response").hide();
 
+   if($('#answer_text').val() ==  "") 
+   $('#submit_answer').attr('disabled', true);
```

Comment:
For more readability, please either (1) put the `if (x) ...;` all on one line, or (2) wrap the second line in curly braces. This is common JS practice (see, for example, https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Code_formatting#Code_formatting -- the first section on Curly Braces; or the '1TBS' section on http://www.2ality.com/2013/01/brace-styles.html).